mmcv/cnn/rfsearch/operator.py

```diff
@@ -0,0 +1,168 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import copy
+import logging
+from typing import Dict
+
+import torch
+import torch.nn as nn
+from torch import Tensor
+
+from .utils import expands_rate, value_crop
+
+logger = logging.getLogger('Operators')
+logger.setLevel(logging.INFO)
+
+
+class ConvRFSearchOp(nn.Module):
+
+    def __init__(self, op_layer: nn.Module, global_config: Dict = {}):
```

Comment:
Is better to set the default `global_confg` as None.

Reply:
Changing the global_config to None cannot pass the code checking. So we keep it unchanged for now.